hudi-cli/src/main/java/org/apache/hudi/cli/commands/BootstrapCommand.java

```diff
@@ -72,8 +72,14 @@ public String bootstrap(
           help = "Class for Full bootstrap input provider") final String fullBootstrapInputProvider,
       @ShellOption(value = {"--schemaProviderClass"}, defaultValue = "",
           help = "SchemaProvider to attach schemas to bootstrap source data") final String schemaProviderClass,
-      @ShellOption(value = {"--payloadClass"}, defaultValue = "org.apache.hudi.common.model.OverwriteWithLatestAvroPayload",
-          help = "Payload Class") final String payloadClass,
+      @ShellOption(value = {"--payloadClass"}, defaultValue = "",
+          help = "Payload Class (deprecated). Use merge-mode for overwrite or event time merging.") final String payloadClass,
+      @ShellOption(value = {"--merge-mode", "--record-merge-mode"}, defaultValue = "",
+          help = "Merge mode to use. 'EVENT_TIME_ORDERING', 'OVERWRITE_WITH_LATEST', or 'CUSTOM' if you want to set a merge strategy") final String recordMergeMode,
+      @ShellOption(value = {"--merger-strategy", "--record-merger-strategy"}, defaultValue = "",
```

Comment:
Do we have any validation to check that the merge strategy ID is only set for `CUSTOM` merge mode?